tessellation/src/path_stroke.rs

```diff
@@ -616,20 +615,21 @@ impl<'l, Output: 'l + GeometryBuilder<Vertex>> StrokeBuilder<'l, Output> {
                     back_vertex
                 )
             }
+            LineJoin::MiterClip => {
+                self.tessellate_miter_clip_join(
+                    prev_tangent,
+                    next_tangent,
+                    front_side,
+                    back_vertex,
+                    normal
+                )
+            }
             // Fallback to Miter for unimplemented line joins
             _ => {
-                let v = add_vertex!(
-                    self,
-                    Vertex {
-                        position: self.current,
-                        normal: front_normal,
-                        advancement: self.length,
-                        side: front_side,
-                    }
-                );
-                self.prev_normal = normal;
-
-                (v, v)
+                self.tessellate_miter_join(
+                    front_side,
+                    normal
```

Comment:
Oops actually there was a bug here: this should be front_normal instead of normal, I'll land a quick fix.